### PR TITLE
Extra env

### PR DESCRIPTION
**Pull request by superseb**

### CODE REVIEW

**Comment by alena1108** [comment]
@superseb @galal-hussein could you provide a link to issue or explanation for the feature

**Comment by alena1108** [comment]
rancher/rancher#13816

File: apis/management.cattle.io/v3/rke_types.go [hunk 1]
```diff
@@ -229,6 +229,8 @@ type BaseService struct {
 	ExtraArgs map[string]string `yaml:"extra_args" json:"extraArgs,omitempty"`
 	// Extra binds added to the nodes
 	ExtraBinds []string `yaml:"extra_binds" json:"extraBinds,omitempty"`
+	// Extra environment variables added to the services
+	ExtraEnv []string `yaml:"extra_env" json:"extraEnv,omitempty"`
```
Comment (alena1108) [inline apis/management.cattle.io/v3/rke_types.go]: @superseb could we elaborate on how those env vars are gonna be used. @galal-hussein suggested `this is to provide extra env variable to the docker container running kubernetes service`

**Comment by superseb** [comment]
Its updated

**Pull request merged**